artman/conductors/cloudtasks_conductor.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Artman conductor to claim and execute remote tasks.'''
+
+import base64
+import io
+import os
+import subprocess
+import time
+import uuid
+
+from oauth2client.client import GoogleCredentials
+from googleapiclient.discovery import build_from_document
+
+from artman.cli import main
+from artman.utils.logger import logger
+
+
```

Comment:
At a high level, it makes me a bit squeamish that we are implementing our own task runner. Why not use an existing one (e.g. [celery](http://www.celeryproject.org/))?